source/services/MonitoringService.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentType } from "react";
+import env from "react-native-config";
+
+import SentryMonitoringService from "./SentryMonitoringService";
+
+export interface MonitoringService {
+  init: () => void;
+  sendError: (Error: Error | string | null) => void;
+  wrap: (component: ComponentType<unknown>) => ComponentType<unknown>;
+}
+
+const nullMonitoringService: MonitoringService = {
+  init: () => undefined,
+  sendError: () => undefined,
+  wrap: (component) => component,
+};
```

Comment:
Could we move `nullMonitoringService` to a separate file? I think it's more concise to have concrete implementations in separate files even if it's a "fallback" implementation.

Reply:
Fixed